src/Greeting.sol

```diff
@@ -5,8 +5,6 @@ pragma solidity ^0.8.23;
 struct Greeting {
     uint64 sourceChainId;   // The source chain ID of the greeting
     uint256 timestamp;      // The timestamp of the greeting
-    uint256 fee;            // The portal fee for the greeting
```

Comment:
no need to track fee in the greeting directly, it's part of the XMsg, not the greeting.